tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,18 +123,467 @@ def test_rgw_obc_quota(
             logger.info(f"New quota {new_quota_str} got applied!!")
 
-    @polarion_id("OCS-6178")
+        # Decrease maxObjects below current usage and verify writes are blocked
+        decreased_quota = 2
+        decreased_quota_str = (
+            f'{{"spec": {{"additionalConfig":{{"maxObjects": "{decreased_quota}"}}}}}}'
+        )
+        cmd = f"patch obc {bucket_name} -p '{decreased_quota_str}' -n openshift-storage --type=merge"
+        OCP().exec_oc_cmd(cmd)
+        logger.info(
+            f"Decreased maxObjects to {decreased_quota} (below current usage) on obc {bucket_name}"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)